src/test/utils.tsx

```diff
@@ -3,30 +3,12 @@ import { render as rtlRender } from "@testing-library/react";
 import { Provider as IntlProvider } from "../locale";
 import { Provider as ThemeProvider } from "../theme";
 import { MockedProvider as MockedGqlProvider, MockedResponse } from "@apollo/client/testing";
-import { LANGUAGE } from "@reearth/locale/queries";
 
-const queryMocks: readonly MockedResponse<Record<string, any>>[] | undefined = [
-  {
-    request: {
-      query: LANGUAGE,
-      variables: {},
-    },
-    result: {
-      data: {
-        me: {
-          id: "whatever",
-          name: "mock",
-          email: "mock@example.com",
-          lang: "en",
-          myTeam: { id: "whatever", name: "sample", __typename: "Team" },
-          auths: ["auth0"],
-          __typename: "User",
-        },
-      },
-    },
-  },
-];
-const render = (ui: React.ReactElement, { ...renderOptions } = {}) => {
+const render = (
+  ui: React.ReactElement,
+  queryMocks?: readonly MockedResponse<Record<string, any>>[],
+  { ...renderOptions } = {},
+) => {
```

Comment:
On an unrelated topic, I don't like this helper function for unit testing. It's better to directly import testing-library in each test case, so that we can see at a glance what context the component requires just by looking at the test code.